pkg/vpcmodel/subnetsConnectivity.go

```diff
@@ -252,14 +255,45 @@ func (v *VPCsubnetConnectivity) computeAllowedConnsCombined() error {
 			v.AllowedConnsCombined[src][dst] = combinedConns
 		}
 	}
+	return nil
+}
 
+func (v *VPCsubnetConnectivity) computeStatefulConnections() error {
+	for src, endpointConns := range v.AllowedConnsCombined {
+		for dst, conns := range endpointConns {
+			if conns.IsEmpty() {
+				continue
+			}
+			dstObj := v.cloudConfig.NameToResource[dst.Name()]
+			var otherDirectionConn *common.ConnectionSet = nil
+			switch dstObj.(type) {
+			case NodeSet:
+				otherDirectionConn = v.AllowedConnsCombined[dst][src]
+			case *ExternalNetwork:
```

Comment:
why do we expect `dstObj`  as returned from `NameToResource` to be `ExternalNetwork` ?

Reply:
Why not? Seems we are not on the same page -  perhaps we should take it offline